### ISSUE TITLE
Styles for Poverty Level (STD/GeoServer)

### ISSUE THREAD

**Issue by milafrerichs** [GFDRR/recovery_dashboard#29]
We need two styles for the poverty levels
- [ ] avg %
- [ ] abs numbers

**Comment by avalenzuela00** [comment]
I have both .sld files now. Absolute numbers doesn't really show anything significant. How would you like me to share?

**Comment by ks2693** [comment]
why don't we just show the percent layer (no option to show the absolute)  and in the legend just have % poor layer visible. 

Also, for the legend text, pls change  "avg %" to "% poor per district"

If you click on the data, will it show the data in the attribute table for each district?

**Comment by jj0hns0n** [comment]
This layer is still being pulled from the demo.geonode.org server. it needs to be migrated to the labs geonode and configured in the viewer to use this one. The style is correct? on the labs geonode here http://45.55.174.20/layers/hazard%3Apoverty

**Comment by ks2693** [comment]
Yes, the styling is correct for that one - however i want the categories to
say 0-20%, 20-40%, 40-60%, 60-80%, 80-90%, not 0-0.20, 0.20-0.40 etc etc.

Ana, will you be able to change the class labels in geonode?   -Keiko

**Comment by avalenzuela00** [comment]
I would probably have to create a new field with the real percentages. But yes, it is doable.
